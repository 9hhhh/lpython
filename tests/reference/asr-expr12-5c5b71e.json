{
    "basename": "asr-expr12-5c5b71e",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/expr12.py",
    "infile_hash": "a28e73367996457cb261785aa6032bf884113aab4a9ec1967cd99bbc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-expr12-5c5b71e.stdout",
    "stdout_hash": "9d31675d60e38ae6f3b9a9c2ddf3693b4fb8b1466a05bc4308184d46",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}