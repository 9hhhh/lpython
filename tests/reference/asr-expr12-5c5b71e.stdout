(TranslationUnit (SymbolTable 1 {check: (Function (SymbolTable 3 {_lpython_return_variable: (Variable 3 _lpython_return_variable ReturnVar () () Default (Integer 4 []) Source Public Required .false.), a: (Variable 3 a Local () () Default (Integer 4 []) Source Public Required .false.)}) check [] [(= (Var 3 a) (FunctionCall 1 test () [(ConstantInteger 2 (Integer 4 [])) (ConstantInteger 2 (Integer 4 []))] [] (Integer 4 []) () ()) ()) (= (Var 3 _lpython_return_variable) (Var 3 a) ()) (Return)] (Var 3 _lpython_return_variable) Source Public Implementation ()), test: (Function (SymbolTable 2 {_lpython_return_variable: (Variable 2 _lpython_return_variable ReturnVar () () Default (Integer 4 []) Source Public Required .false.), a: (Variable 2 a In () () Default (Integer 4 []) Source Public Required .false.), b: (Variable 2 b In () () Default (Integer 4 []) Source Public Required .false.)}) test [(Var 2 a) (Var 2 b)] [(= (Var 2 _lpython_return_variable) (BinOp (Var 2 a) Pow (Var 2 b) (Integer 4 []) () ()) ()) (Return)] (Var 2 _lpython_return_variable) Source Public Implementation ())}) [])
